### PR TITLE
Added Dijkstras Algorithm

### PR DESCRIPTION
**Pull request by bagariaraj23**

### CODE REVIEW

File: src/main/java/org/example/Algorithms/dijkstras.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
+/*
+     Write a program to implement Dijkstra's Algorithm. 
+*/
+
+import java.util.*;
+
+class dijkstraClass{
```
Comment (BimalRajGyawali) [inline src/main/java/org/example/Algorithms/dijkstras.java:7]: Please rename class to DijkstraClass to follow _CamelCase_ convention for class names.

**Review by BimalRajGyawali — COMMENTED** [review]
- Filename should be name of public class

File: src/main/java/org/example/Algorithms/dijkstras.java (hunk 1 above)
Comment (bagariaraj23) [reply src/main/java/org/example/Algorithms/dijkstras.java:7]: okay i'll add it right away.

**Review by BimalRajGyawali — COMMENTED** [review]
Filename should be name of public class

**Comment by bagariaraj23** [comment]
hi @BimalRajGyawali , i've made the necessary changes please review them and approve them. thank you!
Also, if there are other requirements let me know.

**Review by BimalRajGyawali — APPROVED** [review]

**Pull request merged**